src/fundamental/tpm2-pcr.h

```diff
@@ -49,3 +49,6 @@ enum {
 
 /* The tag used for EV_EVENT_TAG event log records covering Devicetree blobs */
 #define DEVICETREE_ADDON_EVENT_TAG_ID UINT32_C(0x6c46f751)
+
+/* The tag used for EV_EVENT_TAG event log records covering ucode addons (effectively initrds) */
+#define UCODE_ADDON_EVENT_TAG_ID UINT32_C(0xdac08e1a)
```

Comment:
Please document this briefly in docs/TPM2_PCR_MEASUREMENTS.md, just copy the entry for the devicetree stuff, and make minor adjustments